src/Orchard/ContentManagement/DefaultHqlQuery.cs

```diff
@@ -5,6 +5,8 @@
 using System.Linq;
 using System.Text;
 using NHibernate;
+using NHibernate.Engine;
+using NHibernate.Hql.Ast.ANTLR;
```

Comment:
I forgot to add those changes originally copy pasting changes through GitHub web UI for this PR. For last commit I used a regular Git client so, on compiling I detected this PR was broken because of that.